docker-compose-dev.yml

```diff
@@ -23,18 +24,20 @@ services:
         aliases:
           - apps
 
-
   elm:
     build:
       context: .
-      dockerfile: Dockerfile-elm
+      dockerfile: dockerfiles/dev/Dockerfile-elm
+    container_name: jarbas_elm
     volumes:
-      - "./:/code"
+      - "./jarbas:/code/jarbas"
```

Comment:
When node update its dependencies, it creates a folder in /code/node_modules which is "removed" when docker-compose mount volume. This make the gulp command fail.